.github/workflows/ci.yaml

```diff
@@ -0,0 +1,42 @@
+name: ci
+
+on: [ push, pull_request ]
+
+jobs:
+  build-test:
+    name: Test on PHP ${{ matrix.php-versions }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: [ '7.3', '7.4' ]
```

Comment:
Does it make sense to add php 8.0 here too?

Reply:
Sure, let's do that